tensorboard/webapp/widgets/data_table/data_table_component.scss

```diff
@@ -24,12 +24,23 @@ limitations under the License.
     position: sticky;
     text-align: left;
     top: 0;
+    vertical-align: bottom;
 
     @include tb-dark-theme {
       background-color: map-get($tb-dark-background, background);
     }
   }
 
+  .cell {
+    display: flex;
+    align-items: center;
+  }
+
+  .without-icon {
```

Comment:
Instead of "without-icon" I think an expressive naming is like "extra-right-padding". Especially there is only one css change here.
It's not clear to me when I see "without-icon" in the html